src/cas/flatten_data_to_anndata.py

```diff
@@ -81,11 +82,12 @@ def flatten_cas_object(input_json, anndata_file_path, output_file_path):
     parent_cell_ids = collect_parent_cell_ids(input_json)
 
     with h5py.File(anndata_file_path, "r+") as f:
+        marker_list = np.array(f["var"]["Gene"]["categories"][()], dtype=str)
```

Comment:
What if `Gene` column doesn't exist in `var`? Can we get column name as an optional input parameter from user? If a value is not provided by user, but `Gene` is in the `var`, we can still use it as default value.